deployment/ccip/operation/evm/v1_6/ops_nonce_manager.go

```diff
@@ -98,6 +98,42 @@ var (
 				DescribedTimelockProposals: csOutput.DescribedTimelockProposals,
 			}, nil
 		})
+
+	NonceManagerPreviousRampsUpdatesOp = operations.NewOperation(
+		"NonceManagerPreviousRampsUpdates",
+		semver.MustParse("1.0.0"),
+		"Applies previous ramps updates in NonceManager 1.6 contract on the specified evm chain",
+		func(b operations.Bundle, deps opsutil.ConfigureDependencies, input NonceManagerApplyPreviousRampsUpdatesInput) (opsutil.OpOutput, error) {
+			state := deps.CurrentState
+			e := deps.Env
+			err := input.Validate(e, state)
```

Comment:
Hmm, this is interesting, @kylesmartin. imo, keeping the sequences & especially Operations clean of the pre-req validations and non-on-chain logic, if we can avoid, would make it easier to read & importantly, it would make it less prone to breaking during & save some runtime resources during retries. wdyt?